### PR TITLE
Added Russian localization

### PR DESCRIPTION
**Pull request by syorito-hatsuki**
My team made a free translation for u mod. I hope players be happy to play in their own language :3

Sincerely, Project Manager of the Enthusiasm Dev Team

### CODE REVIEW

File: src/main/resources/assets/alloy_forgery/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "block.alloy_forgery.bricks_forge_controller": "Контроллер кузницы из кирпичей",
+  "block.alloy_forgery.stone_bricks_forge_controller": "Контроллер кузницы из каменных кирпичей",
+  "block.alloy_forgery.deepslate_bricks_forge_controller": "Контроллер кузницы из полированного глубинного сланца",
+  "block.alloy_forgery.end_stone_bricks_forge_controller": "Контроллер кузницы из эндерняковых кирпичей",
+  "block.alloy_forgery.polished_blackstone_forge_controller": "Контроллер кузницы из полированного чернита",
+
+  "tooltip.alloy_forgery.forge_tier": "Уровень: %d",
+  "tooltip.alloy_forgery.fuel_capacity": "Запас Топлива: %d",
```
Comment (BasiqueEvangelist) [inline src/main/resources/assets/alloy_forgery/lang/ru_ru.json:9]: Why is Топлива capitalized?

File: src/main/resources/assets/alloy_forgery/lang/ru_ru.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "block.alloy_forgery.bricks_forge_controller": "Контроллер кузницы из кирпичей",
+  "block.alloy_forgery.stone_bricks_forge_controller": "Контроллер кузницы из каменных кирпичей",
+  "block.alloy_forgery.deepslate_bricks_forge_controller": "Контроллер кузницы из полированного глубинного сланца",
```
[the remark this answers is not in the document]
Comment (syorito-hatsuki) [reply src/main/resources/assets/alloy_forgery/lang/ru_ru.json]: Fixed

File: src/main/resources/assets/alloy_forgery/lang/ru_ru.json (hunk 1 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/alloy_forgery/lang/ru_ru.json]: Guys forgot about text style. Fixed

File: src/main/resources/assets/alloy_forgery/lang/ru_ru.json [hunk 3]
```diff
@@ -1,19 +1,19 @@
 {
   "block.alloy_forgery.bricks_forge_controller": "Контроллер кузницы из кирпичей",
   "block.alloy_forgery.stone_bricks_forge_controller": "Контроллер кузницы из каменных кирпичей",
-  "block.alloy_forgery.deepslate_bricks_forge_controller": "Контроллер кузницы из полированного глубинного сланца",
+  "block.alloy_forgery.deepslate_bricks_forge_controller": "Контроллер кузницы из глубинного сланца",
```
Comment (Noaaan) [inline src/main/resources/assets/alloy_forgery/lang/ru_ru.json:4]: Isn't this just "Deepslate Forge Controller"? 
It should be "Deepslate Bricks Forge Controller".

**Comment by syorito-hatsuki** [comment]
The translator says that he did not sleep and that is why he made so many mistakes. They asked not to translate again in this state

**Pull request merged**